frontend/packages/console-shared/src/components/formik-fields/ResourceDropdownField.tsx

```diff
@@ -62,8 +62,8 @@ const ResourceDropdownField: React.FC<ResourceDropdownFieldProps> = ({
           dropDownClassName={cx({ 'dropdown--full-width': fullWidth })}
           onLoad={onLoad}
           resourceFilter={resourceFilter}
-          onChange={(value: string, name) => {
-            props.onChange && props.onChange(value, name);
+          onChange={(value: string, name: string, resource: K8sResourceKind) => {
```

Comment:
```suggestion
          onChange={(value: string, name: string | object, resource: K8sResourceKind) => {
```
Technically, in our case, this is a React Element... but that's also just an object. We don't want to say `string` and get something that is not a `string` that defeats the purpose of TypeScript.